src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/DSASignatureFormat.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    public enum DSASignatureFormat
+    {
+        IeeeP1363FixedFieldConcatenation,
+        Rfc3279DerSequence
```

Comment:
Please add the trailing comma.

Also, please go ahead and write xmldoc for these.